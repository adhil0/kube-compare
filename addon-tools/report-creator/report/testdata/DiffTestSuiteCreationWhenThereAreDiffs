{"Summary":{"ValidationIssuses":{},"NumMissing":0,"UnmatchedCRS":[],"NumDiffCRs":1,"TotalCRs":1,"MetadataHash":"013675dbf39d109d2e17bef23e4786717e5439e5490cf20853af5481f0818c40","patchedCRs":0},"Diffs":[{"DiffOutput":"diff -u -N TEMP/v1_configmap_kubernetes-dashboard_kubernetes-dashboard-settings TEMP/v1_configmap_kubernetes-dashboard_kubernetes-dashboard-settings\n--- TEMP/v1_configmap_kubernetes-dashboard_kubernetes-dashboard-settings\tDATE\n+++ TEMP/v1_configmap_kubernetes-dashboard_kubernetes-dashboard-settings\tDATE\n@@ -2,6 +2,6 @@\n kind: ConfigMap\n metadata:\n   labels:\n-    k8s-app: kubernetes-dashboardfunction was called successfully from different file\n+    k8s-app: kubernetes-dashboard\n   name: kubernetes-dashboard-settings\n   namespace: kubernetes-dashboard\n","CorrelatedTemplate":"cm.yaml","CRName":"v1_ConfigMap_kubernetes-dashboard_kubernetes-dashboard-settings"}]}
