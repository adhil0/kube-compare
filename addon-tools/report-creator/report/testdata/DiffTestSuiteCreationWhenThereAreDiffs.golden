<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Comparison results of known valid reference configuration and a set of specific cluster CRs" tests="3" failures="1" errors="0" skipped="0" TIME>
	<testsuite tests="1" failures="1" skipped="0" TIME name="Detected Differences Between Cluster CRs and Expected CRs" TIME>
		<properties></properties>
		<testcase classname="Matching Reference CR: cm.yaml" name="CR: v1_ConfigMap_kubernetes-dashboard_kubernetes-dashboard-settings" TIME>
			<properties></properties>
			<failure message="Differences found in CR: v1_ConfigMap_kubernetes-dashboard_kubernetes-dashboard-settings, Compared To Reference CR: cm.yaml" type="Difference">diff -u -N TEMP/v1_configmap_kubernetes-dashboard_kubernetes-dashboard-settings TEMP/v1_configmap_kubernetes-dashboard_kubernetes-dashboard-settings&#xA;--- TEMP/v1_configmap_kubernetes-dashboard_kubernetes-dashboard-settings&#x9;DATE&#xA;+++ TEMP/v1_configmap_kubernetes-dashboard_kubernetes-dashboard-settings&#x9;DATE&#xA;@@ -2,6 +2,6 @@&#xA; kind: ConfigMap&#xA; metadata:&#xA;   labels:&#xA;-    k8s-app: kubernetes-dashboardfunction was called successfully from different file&#xA;+    k8s-app: kubernetes-dashboard&#xA;   name: kubernetes-dashboard-settings&#xA;   namespace: kubernetes-dashboard&#xA;</failure>
		</testcase>
	</testsuite>
	<testsuite tests="1" failures="0" skipped="0" TIME name="Reference validation" TIME>
		<properties></properties>
		<testcase classname="" name="No reference validation issues: All expected CRs exist in the cluster" TIME>
			<properties></properties>
		</testcase>
	</testsuite>
	<testsuite tests="1" failures="0" skipped="0" TIME name="Unmatched Cluster Resources" TIME>
		<properties></properties>
		<testcase classname="" name="All Cluster CRs are matched to reference CRs " TIME>
			<properties></properties>
		</testcase>
	</testsuite>
</testsuites>